src/includes/class_reservationPermissions.php

```diff
@@ -109,29 +122,32 @@ public function checkBuilding($id = null){
     }
   }
 
-  public function checkRoom($id = null){
+  // checkRoom
+  // ==================================================
+  // Name : Tracy McCormick
+  //
+  // Description:
+  // Function requires room ID.
+  // Check and sees if the room ID is listed in the reservePermission table
+  // returns true if found false otherwise
+  public function checkRoom($roomID = null){
     try {
         // test to see if Id is present and valid
-        if(!isnull($id) && !$this->validate->integer($id)){
+        if(!isnull($roomID) && !$this->validate->integer($roomID)){
```

Comment:
+1 my previous comment. 

You need to check it up above as well (see previous comment)